tests/terraform/graph/variable_rendering/expected_data.py

```diff
@@ -31,7 +31,7 @@
             "version": ["1.19"],
             "vpc_config": {
                 "security_group_ids": ["aws_security_group.master.id"],
-                "subnet_ids": "Tree('full_splat_expr_term', ['aws_subnet.eks', 'id'])"
+                "subnet_ids": "aws_subnet.eks[*].id"
```

Comment:
@gruebel Very cool that we now support it. How do we parse it as graph node attributes? As a string?

Reply:
our Transformer in the `hcl2` lib was missing a method, that's why it looked so weird. let me check, how it looks like in the graph node 🧐

Reply:
@nimrodkor currently it is just a string